spacemacs/packages.el

```diff
@@ -1677,14 +1677,19 @@ If ARG is non nil then `ag' and `pt' and ignored."
           (call-interactively (spacemacs//helm-projectile-do-search-find-tool
                                tools))))
 
+      (setq helm-projectile-sources-list '(helm-source-projectile-projects
+                                           helm-source-projectile-files-list
+                                           helm-source-recentf
+                                           helm-source-pp-bookmarks
+                                           helm-source-file-cache))
```

Comment:
The problem is not for saving key bindings, but for reducing cognitive load. We have an all in one command, it would make it easier for user to learn and use it, as suggested by @chrisbarrett in the this issue: https://github.com/syl20bnr/spacemacs/issues/772. And for Vim users, dealing multiple sources for CtrlP or Unite is common @ReneFroger: do you agree? Could give some input?

Also, the point of this PR is giving the functionality like CtrlP. If we just reduce to one source per command, basically we should not discuss adding CtrlP in the first place!